src/hooks/useMellowLPVaults/utils.ts

```diff
@@ -0,0 +1,30 @@
+
+
+type MellowLpVaultAddresses = {
+    voltzVaultAddress: string;
+    erc20RootVaultAddress: string;
+    erc20RootVaultGovernanceAddress: string;
+};
+
+export const getMellowLPAddresses = (): MellowLpVaultAddresses[] => {
```

Comment:
pass process.env.REACT_APP_MELLOW_VAULTS as param to this function. 

Write tests please!
```suggestion
export const getMellowLPAddresses = (mellowVaults?: string | null): MellowLpVaultAddresses[] => {
```

Reply:
Resolved. 
https://github.com/Voltz-Protocol/voltz-ui/pull/295/commits/f78903feec31bb3511c3a5f0608a6b2c76d8fca6